Gemfile

```diff
@@ -35,9 +35,22 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+# views 
+gem 'haml-rails', '~> 1.0'
+# Authentication
+gem 'devise', '~> 4.5'
+
+
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
+  # configuration of local environment variables
+  gem 'dotenv-rails'
+  # console
+  gem 'awesome_print', '~> 1.8'
+  # debug
+  gem 'pry-nav', '~> 0.2.4'
+  gem 'pry-rails'
 end
 
 group :development do
```

Comment:
the same

Reply:
ordered gems except the version of the rails for visibility